company/strategy.md

```diff
@@ -52,15 +52,15 @@ Our [values](../company#values) help shape the culture we want to build, both in
 
 #### 2022
 
-There's four pillars of the product we're investing in:
-1. Node-RED management
-1. Edge Deployments and management
-1. Collaboration features
-1. Enterprise and compliance
+There are four pillars of the product we're investing in:
+1. Provide the best Node-RED experience.
+1. Enable users to run Node-RED at the edge.
+1. Enable teams to collaborate when working with Node-RED.
+1. Meet the compliance needs of the enterprise.
```

Comment:
I think more than compliance though. RBAC, SSO, Audit logging, MFA, etc all seem more "enterprise" then "compliance"?

Reply:
They're all things that enterprise customers want in order to meet their compliance needs though, MFA is about securing your system and having a secure system is needed to meet compliance needs, compliance isn't only about regulatory requirements.